src/tests/test_plot.py

```diff
@@ -55,3 +57,45 @@ def test_boxplot_stats(chinook_db):
     result = plot._boxplot_stats(con, "Invoice", "Total")
 
     assert DictOfFloats(result) == DictOfFloats(expected[0])
+
+
+def test_boxplot_stats_exception(chinook_db):
+    con = duckdb.connect(database=":memory:")
+    con.execute("INSTALL 'sqlite_scanner';")
+    con.execute("LOAD 'sqlite_scanner';")
+    con.execute(f"CALL sqlite_attach({chinook_db!r});")
+
+    res = con.execute("SELECT * FROM Invoice")
+    X = res.df().Total
+    cbook.boxplot_stats(X)
+    with pytest.raises(
+        BaseException, match="whis must be a float or list of percentiles.*"
+    ):
+        plot._boxplot_stats(
+            con, "Invoice", "Total", "Not a float or list of percentiles whis param"
+        )
+
+
+@pytest.mark.parametrize(
+    "cell, error_type, error_message",
+    [
+        [
+            "%sqlplot histogram --table data.csv --column age --table data.csv",
+            ValueError,
```

Comment:
What about TypeErrors?
I also saw more instances of those are those tested?

Reply:
I think during the invocation of `%sqlplot histogram` or `%sqlplot boxplot` we only throw `ValueError` now, https://github.com/ploomber/jupysql/blob/8c9ce8c8b04c56916bab07e3bc05bc6798d477c3/src/sql/plot.py#L382

We only have [one place](https://github.com/tonykploomber/jupysql/blob/master/src/sql/magic.py#L362) to raise TypeError, although we already had a test case to cover it, I made [a patch](https://github.com/ploomber/jupysql/pull/106/commits/21306b6f4d3f0deea8396776496bfba97ab6fca6) to check it's TypeError instance

Reply:
Ok